app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -306,6 +329,24 @@ private void updateViews() {
         }
     }
 
+    private void updateOverviewVideo() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```

Comment:
you don't need this version check, `mVideoBanner` will only be non-null if this version condition is met